Move flag to configmap (#1540)
zalando/postgres-operator: docs/user.md

other roles.

To define the secrets for the users in a different namespace than that of the cluster,

> one can use the flag `EnableCrossNamespaceSecret` and declare the namespace for the

Reviewer: ```suggestion
set `enable_cross_namespace_secret` to `true` and declare the namespace for the
```
